### ISSUE TITLE
作者发布的aishell 模型无法去restore，retrain 

### ISSUE THREAD

**Issue by fangmingbnu** [PaddlePaddle/DeepSpeech#195]
当我尝试去restore主页上的那个aishell模型的时候，出现错误。这是我的命令，CUDA_VISIBLE_DEVICES=0 python -u train.py --batch_size=16 --trainer_count=1 --num_passes=10 --num_proc_data=16 --num_conv_layers=2 --num_rnn_layers=3 --rnn_layer_size=1024 --num_iter_print=100 --learning_rate=4e-5 --max_duration=27.0 --min_duration=0.0 --test_off=False --use_sortagrad=True --use_gru=True --use_gpu=True --is_local=True --share_rnn_weights=False --train_manifest='data/aishell/manifest.train' --dev_manifest='data/aishell/manifest.dev' --mean_std_path='data/aishell/mean_std.npz' --vocab_path='data/aishell/vocab.txt' --output_model_dir='diymodel/aishell/' --augment_conf_path='conf/augmentation.config' --specgram_type='linear' --shuffle_method='batch_shuffle_clipped' --init_model_path='models/aishell/params.tar.gz'。

错误提示如下：
-----------  Configuration Arguments -----------
augment_conf_path: conf/augmentation.config
batch_size: 16
dev_manifest: data/aishell/manifest.dev
init_model_path: models/aishell/params.tar.gz
is_local: 1
learning_rate: 4e-05
max_duration: 27.0
mean_std_path: data/aishell/mean_std.npz
min_duration: 0.0
num_conv_layers: 2
num_iter_print: 100
num_passes: 10
num_proc_data: 16
num_rnn_layers: 3
output_model_dir: diymodel/aishell/
rnn_layer_size: 1024
share_rnn_weights: 0
shuffle_method: batch_shuffle_clipped
specgram_type: linear
test_off: 0
train_manifest: data/aishell/manifest.train
trainer_count: 1
use_gpu: 1
use_gru: 1
use_sortagrad: 1
vocab_path: data/aishell/vocab.txt
------------------------------------------------
I0327 18:03:09.554020 23876 Util.cpp:166] commandline:  --use_gpu=1 --rnn_use_batch=True --log_clipping=True --trainer_count=1 
[INFO 2018-03-27 18:03:11,308 layers.py:2714] output for __conv_0__: c = 32, h = 81, w = 54, size = 139968
[INFO 2018-03-27 18:03:11,309 layers.py:3282] output for __batch_norm_0__: c = 32, h = 81, w = 54, size = 139968
[INFO 2018-03-27 18:03:11,310 layers.py:7454] output for __scale_sub_region_0__: c = 32, h = 81, w = 54, size = 139968
[INFO 2018-03-27 18:03:11,310 layers.py:2714] output for __conv_1__: c = 32, h = 41, w = 54, size = 70848
[INFO 2018-03-27 18:03:11,310 layers.py:3282] output for __batch_norm_1__: c = 32, h = 41, w = 54, size = 70848
[INFO 2018-03-27 18:03:11,311 layers.py:7454] output for __scale_sub_region_1__: c = 32, h = 41, w = 54, size = 70848
I0327 18:03:15.383965 23876 GradientMachine.cpp:94] Initing parameters..
I0327 18:03:17.461269 23876 GradientMachine.cpp:101] Init parameters done.
F0327 18:03:20.693872 23876 TensorApply.h:126] Check failed: lhs_.getWidth() == rhs_.getWidth() (8876032 vs. 8804352) 
*** Check failure stack trace: ***
    @     0x7fc5aa6570bd  google::LogMessage::Fail()
    @     0x7fc5aa659408  google::LogMessage::SendToLog()
    @     0x7fc5aa656bcb  google::LogMessage::Flush()
    @     0x7fc5aa65a2de  google::LogMessageFatal::~LogMessageFatal()
    @     0x7fc5aa7b49c8  paddle::adamApply()
    @     0x7fc5aa7a3b59  paddle::AdamParameterOptimizer::update()
    @     0x7fc5aa7a3f82  paddle::OptimizerWithGradientClipping::update()
    @     0x7fc5aa78fe1e  paddle::SgdThreadUpdater::updateImpl()
    @     0x7fc5aa62c7a1  ParameterUpdater::update()
    @     0x7fc5aa0d5ff7  _wrap_ParameterUpdater_update
    @           0x4bc3fa  PyEval_EvalFrameEx
    @           0x4c136f  PyEval_EvalFrameEx
    @           0x4b9ab6  PyEval_EvalCodeEx
    @           0x4c16e7  PyEval_EvalFrameEx
    @           0x4b9ab6  PyEval_EvalCodeEx
    @           0x4c16e7  PyEval_EvalFrameEx
    @           0x4b9ab6  PyEval_EvalCodeEx
    @           0x4c1e6f  PyEval_EvalFrameEx
    @           0x4b9ab6  PyEval_EvalCodeEx
    @           0x4c1e6f  PyEval_EvalFrameEx
    @           0x4b9ab6  PyEval_EvalCodeEx
    @           0x4eb30f  (unknown)
    @           0x4e5422  PyRun_FileExFlags
    @           0x4e3cd6  PyRun_SimpleFileExFlags
    @           0x493ae2  Py_Main
    @     0x7fc5e416e830  __libc_start_main
    @           0x4933e9  _start
    @              (nil)  (unknown)
Aborted (core dumped)
不知道是不是什么参数设置错了，请教大家，谢谢！

**Comment by fangmingbnu** [comment]
问题解决了，我在训练的时候没有用作者模型自带的“mean_std.npz“ 和”vacab.txt"文件，所以无法去读取这个文件。现在换成自带的，问题解决了。
